test/unit/geo/projection/far_z.test.js

```diff
@@ -0,0 +1,66 @@
+import {test} from '../../../util/test.js';
+import {farthestPixelDistanceOnPlane, furthestPixelDistanceOnSphere} from '../../../../src/geo/projection/far_z.js';
```

Comment:
Just realized I introduced mixed naming here: `farthestPixelDistanceOnPlane` and `furthestPixelDistanceOnSphere`. Could you replace that in this PR for `farthestPixelDistanceOnSphere`?

Reply:
Sure! I was about to ask you if it was intentional. :D I'll rename it then.